frontend/package.json

```diff
@@ -145,9 +145,8 @@
     "postcss": "^8.4.12"
   },
   "engines": {
-    "node": "^14.17.0 || >= 16.0.0",
-    "npm": ">=6.0.0 ",
```

Comment:
Node 18.0.0 already comes with NPM 8.6.0 installed, so this condition is not needed anymore (fulfilled when you install node 18 or greater version).